lib/at_monitor/at_monitor.c

```diff
@@ -107,6 +120,16 @@ static int at_monitor_sys_init(void)
 {
 	int err;
 
+#if CONFIG_AT_MONITOR_USE_DEDICATED_WORKQUEUE
+	const struct k_work_queue_config cfg = {.name = "at_monitor_work_q"};
+
+	k_work_queue_start(&at_monitor_work_q,
+			   at_monitor_work_q_stack,
+			   K_THREAD_STACK_SIZEOF(at_monitor_work_q_stack),
+			   K_LOWEST_APPLICATION_THREAD_PRIO,
```

Comment:
Should we keep the current priority that we have from the system work queue? I'm trying to say that this priority should probably be higher than most libraries.

Reply:
Agree, I changed it to `-1` by default